esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -134,15 +135,23 @@ define([
                 origOnOpen.apply(context.desprayTooltipDialog, arguments);
 
                 if (!context.desprayIPSelect.initalized) {
+                    var pathSepChar = "/";
                     context.desprayIPSelect.init({
                         DropZoneMachines: true,
                         callback: function (value, row) {
+                            var path = targetRow.machine.Directory.indexOf("\\");
                             targetRow.machine.Name = value
                             targetRow.machine.Netaddress = value
                             if (context.desprayTargetPath) {
                                 context.desprayTargetPath._dropZoneTarget = targetRow;
                                 context.desprayTargetPath.defaultValue = "/"
-                                context.desprayTargetPath.loadDropZoneFolders();
+                                context.desprayTargetPath.loadDropZoneFolders(pathSepChar);
```

Comment:
Made a change as I went from Windows back to Linux the `\` remained. Only calling once based on environment type.